commit 3a78e399f36ed70e5b0d9e094fc9cb05c0b0f945
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Tue Sep 25 16:57:40 2018 +0100

    staging: vc04_services: Fixup vchiq-mmal include ordering
    
    There were dependencies on including the headers in the correct
    order. Fix up the headers so that they include the other
    headers that they depend on themselves.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

Index: linuxtest/drivers/staging/vc04_services/vchiq-mmal/mmal-msg.h
===================================================================
--- linuxtest.orig/drivers/staging/vc04_services/vchiq-mmal/mmal-msg.h
+++ linuxtest/drivers/staging/vc04_services/vchiq-mmal/mmal-msg.h
@@ -38,6 +38,7 @@
 #include "mmal-msg-common.h"
 #include "mmal-msg-format.h"
 #include "mmal-msg-port.h"
+#include "mmal-vchiq.h"
 
 enum mmal_msg_type {
 	MMAL_MSG_TYPE_QUIT = 1,
Index: linuxtest/dummy/rpi_1419_3a78e399f36ed70e5b0d9e094fc9cb05c0b0f945.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1419_3a78e399f36ed70e5b0d9e094fc9cb05c0b0f945.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
